soql-analyzer/src/main/scala/com/socrata/soql/SoQLAnalyzer.scala

```diff
@@ -844,8 +848,28 @@ private class Merger[T](andFunction: MonomorphicFunction[T]) {
 
   private def hasWindowFunction(a: Analysis): Boolean = {
     a.selection.exists {
-      case (_, fc: com.socrata.soql.typed.FunctionCall[_, _]) =>
-        fc.window.nonEmpty
+      case (_, expr) =>
+        hasWindowFunction(expr)
+      case _ =>
+        false
+    }
+  }
+
+  private def hasWindowFunction(e: CoreExpr[_, _]): Boolean = {
+    e match {
+      case fc: com.socrata.soql.typed.FunctionCall[_, _] =>
+        fc.window.nonEmpty || fc.parameters.exists(hasWindowFunction)
+      case _ =>
+        false
+    }
+  }
+
+  private def hasLiteral(e: CoreExpr[_, _]): Boolean = {
+    e match {
+      case fc: com.socrata.soql.typed.FunctionCall[_, _] =>
+        fc.window.nonEmpty || fc.parameters.exists(hasLiteral)
```

Comment:
Why does "has a window clause" make "hasLiteral" true?

Reply:
No it doesn't.  It was a copy paste error.